### DISCUSSION TITLE
List items and new lines

### DISCUSSION THREAD

**Discussion by jglueckstein** [nvim-telekasten/telekasten.nvim#57] [Ideas]
In many markdown editors, such as this github editor, if you create a new line while editing a list item, the new line is automatically a new list item.  I think this functionality in Telekasten would be a time saver and help with "flow".

**Comment by daniel-vera-g** [comment]
Auto bullets, numbering and co is indeed helpful, but I'm not sure whether it should be implemented in Telekasten itself.
Plugins like [vim-markdown](https://github.com/plasticboy/vim-markdown) or [bullets.vim](https://github.com/dkarter/bullets.vim) already have this feature. With the later one, you could even set `telekasten` as a filetype.

**Reply by nathanbraun** [reply]
Yeah I set up bullets.vim with telekasten and it seems to be working great.

**Reply by jglueckstein** [reply]
Have you guys seen this one? [mkdnflow.nvim](https://github.com/jakewvincent/mkdnflow.nvim)

**Reply by daniel-vera-g** [reply]
Seems useful, thanks for sharing. Before Telekasten I used [wiki.vim](https://github.com/lervag/wiki.vim) which is pretty similar. Will test how
it works together with Telekasten.
Had the experience, that using too much plugins made problems as they conflicted among each other(Like who inserts bullets)

**Comment by jglueckstein** [comment]
I think I spoke too soon.  I was thinking about vimwiki behaviors that I was missing (I used markdown format with vimwiki btw).  I thought that a lot of those behaviors came from vimwiki itself.  When I took a look back at my old vim config, I found that a lot of those behaviors came from [vim-markdown](https://github.com/plasticboy/vim-markdown)

Thanks for the replies and I will look into [bullets.vim](https://github.com/dkarter/bullets.vim)
